executor/src/core/storage-services/Holochain/HolochainService.ts

```
@@ -130,7 +127,7 @@ export default class HolochainService {
         await HOLOCHAIN_SERVICE.shutdown();
     }
 
-    async unpackDna(dnaPath: string): Promise<String> {
+    async unPackDna(dnaPath: string): Promise<String> {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)